doc/nrf/ug_thread_power_consumption.rst

```diff
@@ -0,0 +1,100 @@
+* For the SSED case, it will regularly switch to radio receive mode.
+
+The PPK2 is used to collect power consumption data during at least 60 seconds.
+Afterwards, the CSV data is parsed with a script to produce the below results.
+
+.. _thread_power_consumption_data:
+
+Power consumption data
+**********************
+
+The following tables show the power consumption measured with the configuration used.
+
+.. tabs::
+
+   .. group-tab:: Thread 1.3 SED
+
+      .. table:: Configuration
+
+         +--------------------------+-------+
+         | Parameter                | Value |
+         +==========================+=======+
+         | Supply voltage [V]       |   3.0 |
+         +--------------------------+-------+
+         | Transmission power [dBm] |     0 |
+         +--------------------------+-------+
+         | Polling period [ms]      |  1000 |
+         +--------------------------+-------+
+
+      .. table:: Power consumption
+
+         +-------------------------------+-------------------+
+         | Parameter                     | nrf5340dk_nrf5340 |
+         +===============================+===================+
+         | Average sleep current [μA]    |              4.02 |
+         +-------------------------------+-------------------+
+         | Average data poll charge [μC] |             20.11 |
+         +-------------------------------+-------------------+
+         | Total charge per minute [mC]  |              1.57 |
+         +-------------------------------+-------------------+
+
+   .. group-tab:: Thread 1.3 SSED
+
+      .. table:: Configuration
+
+         +-------------------------------+-------+
+         | Parameter                     | Value |
+         +===============================+=======+
+         | Supply voltage [V]            |   3.0 |
+         +-------------------------------+-------+
+         | Transmission power [dBm]      |     0 |
+         +-------------------------------+-------+
+         | CSL period [ms]               |  1000 |
+         +-------------------------------+-------+
+         | CSL timeout [s]               |    20 |
+         +-------------------------------+-------+
+         | Parent's CSL uncertainty [μs] |  ±200 |
+         +-------------------------------+-------+
+         | Parent's CSL accuracy [ppm]   |    ±5 |
+         +-------------------------------+-------+
+
+      .. table:: Power consumption
+
+         +---------------------------------+-------------------+
+         | Parameter                       | nrf5340dk_nrf5340 |
+         +=================================+===================+
+         | Average sleep current [μA]      |              4.02 |
+         +---------------------------------+-------------------+
+         | Average CSL receive charge [μC] |             14.31 |
+         +---------------------------------+-------------------+
+         | Total charge per minute [mC]    |              1.25 |
```

Comment:
This is without any data exchange or some? In general 25% better than SED?

Reply:
That's true, but with some trick... I'll make more clear that the statistics are for one minute of "just listening".

The nRF5 SDK Battery life estimation white paper proposed a few scenarios with different transmission rates (30 min, 10 min...) and frame sizes, but the battery life was impacted by only about a 3%. The only big impact was for the case of 1 min rate, with a 25% decrease. So the idea is to provide just sleepy energy usage plus energy required for a transmission (to be added here later on).

The total energy used by the SSED during a minute is lower than the SED case, but only because the CSL timeout is adjusted to 20s, which is not the default. The reason is that the parent reports 5 pmm accuracy, which makes the receive window duration increase over time as long as there is no synchronization.